From d922234c4970d716e3e2a44298efa4029f607147 Mon Sep 17 00:00:00 2001
From: Longjun Luo <luolongjuna@gmail.com>
Date: Fri, 2 Dec 2022 12:32:07 +0800
Subject: [PATCH] uprobe: enable CONFIG_UPROBES_SUPPORT_PC_ALTER by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: d922234c4970d716e3e2a44298efa4029f607147
Modified-by-SEL: No


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I61AXT
CVE: N/A

Signed-off-by: Longjun Luo <luolongjuna@gmail.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/Kconfig b/arch/Kconfig
index cb7d33051abc..f6d455c5a897 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -157,6 +157,7 @@ config UPROBES
 	    application. )
 
 config UPROBES_SUPPORT_PC_ALTER
+	bool "add support for UPROBE to modify the PC register"
 	def_bool n
 	depends on UPROBES
 	help
-- 
2.33.0

